src/simulation/m_bubbles.fpp

```diff
@@ -230,16 +252,18 @@ contains
         !!  @param fcgas Current gas sound speed
         !!  @param fntait Tait EOS parameter
         !!  @param fBtait Tait EOS parameter
-    function f_rddot_G(fCpbw, fR, fV, fH, fHdot, fcgas, fntait, fBtait)
+        !!  @param Re_inv_l Inverse Reynolds number (pass in from global parameters)
+    pure elemental function f_rddot_G(fCpbw, fR, fV, fH, fHdot, fcgas, fntait, fBtait, Re_inv_l)
         !$acc routine seq
         real(wp), intent(in) :: fCpbw, fR, fV, fH, fHdot
         real(wp), intent(in) :: fcgas, fntait, fBtait
+        real(wp), intent(in) :: Re_inv_l
```

Comment:
Similar to above, since these are `intent(in)` why was the refactor necessary to make it `pure`?